Library/Homebrew/cmd/install.rb

```diff
@@ -1,3 +1,51 @@
+#:* `install` [`--debug`] [`--env=`<std>|<super>] [`--ignore-dependencies`] [`--only-dependencies`] [`--cc=`<compiler>] [`--build-from-source`|`--force-bottle`] [`--devel`|`--HEAD`] <formula>:
```

Comment:
@maxnordlund mentioned he was using `#:` to make RuboCop happy, so I suspect `#/` raises some kind of warning? To me, both look equally readable and are thus fine.